openstudiocore/src/utilities/core/Finder.hpp

```diff
@@ -103,7 +103,7 @@ namespace openstudio{
     typename std::vector<T>::const_iterator it;
     it = find_if(vec.begin(),vec.end(),finder);
     if (it != vec.end()) {
-      index = it - vec.begin();
+      index = (int)(it - vec.begin());

```

Comment:
Thanks @lefticus 

I didn't want to change the signature of the function, would a static cast be a better solution?  Or incrementing a counter?  I am not trying to say that this is a good function...